resources/analysisTools/snvPipeline/filter_PEoverlap.py

```diff
@@ -393,15 +394,15 @@ def performAnalysis(args):
                         remove_base = decreaseInfo[0]
                         remove_is_reverse = decreaseInfo[1]
 
-                        count_duplicate = decrease_counter(count_duplicate, remove_base, REF)
+                        count_duplicate.update(remove_base == REF)
                         (DP4rf, DP4rr, DP4af, DP4ar) = decreaseDP4(remove_base, remove_is_reverse, REF, ALT, DP4rf, DP4rr, DP4af, DP4ar)
 
             nonREFnonALT = nonREFnonALTrev + nonREFnonALTfwd
             #Format: DP2 -> "reference(forward + reverse), alt(forward + reverse)"
-            supple_dup_str = 'DP2sup=' + ','.join(map(str, count_supple))
-            supple_dup_str += ';DP2dup=' + ','.join(map(str, count_duplicate))

```

Comment:
why did you remove the semicolon in this string append?

Reply:
It looks like a bug, it needs semicolon here.